packages/webapp/.eslintrc.cjs

```diff
@@ -3,9 +3,11 @@ const config = {
 	extends: ["next/core-web-vitals", "plugin:@typescript-eslint/recommended", "prettier"],
 	parser: "@typescript-eslint/parser",
 	parserOptions: {
-		project: "./tsconfig.json",
-	},
-	plugins: ["@typescript-eslint"],
+    project: "./tsconfig.json",
+		"sourceType": "module",
+    "ecmaVersion": "latest",
```

Comment:
What's with the weird indentation here?

Reply:
It doesn't seem to appear on my local repo. 😶

Reply:
It's because of mixed tabs and spaces. We should add `.cjs` and `.mjs` files to the list of those who get formatted by prettier, cand you do that?